core/src/main/java/com/linecorp/armeria/client/retry/RetryingRpcClient.java

```diff
@@ -54,11 +52,23 @@
     }
 
     /**
+     * Creates a new {@link Client} decorator that handles failures of an invocation and retries RPC requests.
+     */
+    public static Function<Client<RpcRequest, RpcResponse>, RetryingRpcClient>
+    newDecorator(RetryStrategy<RpcRequest, RpcResponse> retryStrategy,
+                 int defaultMaxAttempts, long responseTimeoutMillisOfEachReq) {
```

Comment:
Could you add javadoc for `responseTimeoutMillisOfEachReq`?
`@param responseTimeoutMillisOfEachReq response timeout for each request. {@code 0} disables the timeout.`

Reply:
That's good idea~! I will add it.